examples/customization/nginx-config.yaml

```diff
@@ -45,6 +45,7 @@ data:
     if ($new_uri) {
         rewrite ^ $new_uri permanent;
     }
+  lb-method: "" # default is least_conn. Sets the load balancing method for upstreams. See http://nginx.org/en/docs/http/load_balancing.html#nginx_load_balancing_methods
```

Comment:
Change "" to "round_robin".  Replace the link with https://docs.nginx.com/nginx/admin-guide/load-balancer/http-load-balancer/#choosing-a-load-balancing-method